pkg/integrations/v4/dm/emitter_no_register.go

```diff
@@ -1,12 +1,28 @@
 package dm
 
 import (
+	"context"
+
 	"github.com/newrelic/infrastructure-agent/pkg/entity"
 	"github.com/newrelic/infrastructure-agent/pkg/fwrequest"
 	"github.com/newrelic/infrastructure-agent/pkg/integrations/v4/protocol"
 )
 
-func (e *emitter) processDatasetNoRegister(intMetadata protocol.IntegrationMetadata, reqMetadata fwrequest.FwRequestMeta, dataSet protocol.Dataset) {
+// emidDatasetWithEmptyEntity will emit the dataset with an empty entity and entity will be created in the backend
+// through entity synthesis.
+func (e *emitter) emidDatasetWithEmptyEntity(intMetadata protocol.IntegrationMetadata, reqMetadata fwrequest.FwRequestMeta, dataSet protocol.Dataset) {
```

Comment:
Typo `emitDatasetWithEmptyEntity`?

Reply:
nice 👁️  ! fixed in https://github.com/newrelic/infrastructure-agent/pull/1357/commits/ca97c864e751dd398c470803e98f1b3678cf5fff